src/modules/tracker.js

```diff
@@ -1737,6 +1737,7 @@ class Tracker {
    * @param {string} parameters.quizVersionId - Quiz version identifier
    * @param {string} parameters.quizSessionId - Quiz session identifier associated with this conversion event
    * @param {string} parameters.url - Current page url
+   * @param {object[]} parameters.items - List of product item unique identifiers in search results listing
```

Comment:
Can we update this to `parameters.items - List of product item objects` to match recommendations and browse?